apps/i18n/common/en_us.json

```diff
@@ -577,6 +577,13 @@
   "copyStudentsConfirm": "Yes, I want to copy student(s) to be in this current section AND the new section.",
   "copyright": "Copyright",
   "copyrightInfoButton": "View copyright information",
+  "copyright_thanks": "We thank our [donors]({donors_link}), [partners]({partners_link}), our [extended team]({team_link}), and our video cast for their support in creating Code Studio.",
+  "copyright_specialRecognition": "We especially want to recognize the engineers from Amazon, Google, and Microsoft who helped create these materials.",
+  "copyright_artFrom": "Minecraft&#8482; &copy; {current_year} Microsoft. All Rights Reserved.<br/>Star Wars&#8482; &copy; {current_year} Disney and Lucasfilm. All Rights Reserved.<br/>Frozen&#8482; &copy; {current_year} Disney. All Rights Reserved.<br/>Ice Age&#8482; &copy; {current_year} 20th Century Fox. All Rights Reserved.<br/>Angry Birds&#8482; &copy; 2009-{current_year} Rovio Entertainment Ltd. All Rights Reserved.<br/>Plants vs. Zombies&#8482; &copy; {current_year} Electronic Arts Inc. All Rights Reserved.<br/>DreamWorks The Bad Guys &copy; {current_year} DreamWorks Animation LLC. All Rights Reserved.<br/>Paramount Pictures Transformers One &copy; {current_year} Paramount Pictures. All Rights Reserved.",
+  "copyright_codeLicense": "Code.org uses p5.play, which is licensed under [the GNU LGPL 2.1]({gnu_license_link}).",
+  "copyright_poweredByAws": "Amazon Web Services and the “Powered by AWS” logo are trademarks of Amazon.com, Inc. or its affiliates in the United States and/or other countries.",
+  "copyright_builtOnGithub": "Built on GitHub from Microsoft",
+  "copyright_trademark": "&copy; Code.org, {current_year}. Code.org&reg;, the CODE logo, Hour of Code&reg; and {cs_discoveries} are trademarks of Code.org.",
```

Comment:
My understanding is that CS Discoveries is the only one we don't want to be translated since we do have copyright on some translations of Hour of Code such as in Spanish and Portuguese.
For the other copyright notices, I just stick to keeping them translatable if they were before.